src/main/java/datastax/astra/migrate/DiffJobSession.java

```diff
@@ -107,8 +108,9 @@ private void diffAndClear(Map<Row, CompletionStage<AsyncResultSet>> srcToTargetR
             try {
                 Row targetRow = srcToTargetRowMap.get(srcRow).toCompletableFuture().get().one();
                 diff(srcRow, targetRow);
-            } catch (Exception e) {
+            } catch (ExecutionException | InterruptedException e) {
```

Comment:
Should simply leave this as-is as both Execution & Interrupted exceptions are already subclasses of the parent Exception. Throwing it as a runtime exception is good to be left added.